urwid/widget/wimp.py

```diff
@@ -381,11 +435,33 @@ class RadioButton(CheckBox):
     }
     reserve_columns = 4
 
+    @typing.overload
     def __init__(
         self,
         group: MutableSequence[CheckBox],
         label,
-        state: bool | Literal["mixed", "first True"] = "first True",
```

Comment:
`mixed` is incorrect state because by default `has_mixed` is false and change possibility is not public exposed